### DISCUSSION TITLE
Unable to link pull request from a different repo to issue (was previously able to)

### DISCUSSION THREAD

**Discussion by sarahngg** [community/community#19657] [Projects and Issues]
I have an issue-only repo (repo name "oh") that I use to manage other source code repos (repo name "port") like in this [doc](https://docs.github.com/en/enterprise-cloud@latest/repositories/creating-and-managing-repositories/creating-an-issues-only-repository). For issues in the issue-only repo "oh", I was previously able to link pull request from the source code repo in the issues but it does not work anymore. Is this a bug or a breaking change?

The following screenshot is an old issue created from the issue-only repo "oh" created on May 19, 2022
- You can see the issue has a linked PR from the source code repo "port" being checked but greyed out
- The instruction now explicitly mention linking pull request from "this repo"
![image](https://user-images.githubusercontent.com/25860320/176319865-8ebf5184-30b5-41c1-9369-7a3e689b8ae2.png)

Any new issues created, I am not able to search any other PRs in other source code repos in the development section.
![image](https://user-images.githubusercontent.com/25860320/176514255-1f0b90f7-f57b-4e46-aa54-2d1e1de64108.png)

**Comment by timabilov** [comment]
Same thing with organization project.
I am really frustrated.
What is the point of creating organization project where you cannot reference  other repository issues, PRs? 

UPDATE: So, it seems, by default, github suggests you only issues in current repo. So you can "manually" link all the issues and it will work. org/repo#issuenumber
Hope it helps to others.

But it would be good to add PR support as well!
